test/test_yaml.py

```diff
@@ -189,3 +191,14 @@ def test_update_fn(self):
         data = to_yaml(data)
         card.update(data)
         self.assertEqual(card.formatted_name, fn)
+
+    def test_parse_field_with_colon(self):
+        card = create_test_card()
+        fn = 'me myself and i'
+        data = {'Formatted name': fn}
+        data = to_yaml(data)
+        print(khard.helpers.convert_to_yaml("Note", "foo: bar", 0, 5, True))
```

Comment:
Why is there a print here? I think `print()` should be avoided in libraries and tests.

Reply:
Thought I'd removed all of those - sorry! `pytest` hides the outputs if the tests pass, which might have fooled me...